scripts/hotspot.js

```diff
@@ -13,20 +13,25 @@
    * @param  {number} id
    * @param  {boolean} isSmallDeviceCB
    * @param  {H5P.ImageHotspots} parent
+   * @param  {string} icon
+   * @param  {image} iconImage
    */
-  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent) {
+  ImageHotspots.Hotspot = function (config, color, id, isSmallDeviceCB, parent, icon, iconImage) {
     var self = this;
     this.config = config;
     this.visible = false;
     this.id = id;
     this.isSmallDeviceCB = isSmallDeviceCB;
+    // A utility variable so you don't have to always check if there is an iconImage.
+    var iconImageExists = (iconImage == null ? false : true); 
 
     if (this.config.content === undefined  || this.config.content.length === 0) {
       throw new Error('Missing content configuration for hotspot. Please fix in editor.');
     }
-
-    this.$element = $('<div/>', {
-      'class': 'h5p-image-hotspot',
+    
+    // Check if there is an iconImage that should be used instead of fontawesome icons to determine the html element.
+    this.$element = $((iconImageExists ? '<img/>' : '<div/>'), {
+      'class': 'h5p-image-hotspot ' + 'h5p-image-hotspot-' + icon,
```

Comment:
Sure. I've made these changes in https://github.com/nmillin/h5p-image-hotspots/commit/726f575b6f37f1bf6eb992e64c1ebb4527e520c8